File: utilities/src/main/java/org/helpapaw/helpapaw/images/ImageLoader.java
package org.helpapaw.helpapaw.images;

import androidx.annotation.DrawableRes;
import android.widget.ImageView;

/**
 * Created by iliyan on 7/29/16
 */
public interface ImageLoader {

    void load(String url, ImageView imageView, @DrawableRes int placeholder);

    void loadWithRoundedCorners(String url, ImageView imageView, @DrawableRes int placeholder);

}

File: utilities/src/main/java/org/helpapaw/helpapaw/images/ImageLoader.kt
package org.helpapaw.helpapaw.images

import androidx.annotation.DrawableRes
import android.widget.ImageView

/**
 * Created by iliyan on 7/29/16
 */
interface ImageLoader {

    fun load(url: String, imageView: ImageView, @DrawableRes placeholder: Int)

    fun loadWithRoundedCorners(url: String, imageView: ImageView, @DrawableRes placeholder: Int)

}
